docs/tutorial/configure_the_navigation.md

```diff
@@ -0,0 +1,113 @@
+# Configure the navigation
+
+## PlatformUI Navigation Hub
+
+As written in the [PlatformUI technical introduction](https://doc.ez.no/display/DEVELOPER/Extending+eZ+Platform+UI), the Navigation Hub gives access to 4 navigation zones which have a set of navigation items. Each Navigation Item is actually a View in the Navigation Hub which can generate one or more links in the menu. Most Navigation Items can even be seen as a View of a given application route. A Navigation Item View is also responsible for handling its *selected state*. This means that the Navigation Items are notified when the application matches a new route and the view can then decide to react accordingly.
+
+PlatformUI comes with 3 different implementations of Navigation Item. They all generate a link to a route with a given anchor text and they differ by the ability to check if the newly matched route in the application is the route they represent:
+
+- the base implementation is `Y.eZ.NavigationItemView`. When the matched application route changes, it sets its selected state if the navigation item route name matches the name of the new matched route in the application
+- the `Y.eZ.NavigationItemParameterView` implementation adds a check on a route parameter. So to appear selected, the route names must match and a given route parameter should be the same in both the application matched route and in the route the view is representing
+- the `Y.eZ.NavigationItemSubtreeView` also adds a match on a route parameter, but in this case it considers the `id` route parameter and checks whether the matched id in the application route is a descendant of a given Location id.
+
+The default structure of the Navigation Hub is defined in the [Navigation Hub view service attributes](https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/services/ez-navigationhubviewservice.js#L377).
```

Comment:
link should be updated to https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/services/ez-navigationhubviewservice.js#L407